### PR TITLE
feat(pagination): Add tests for pagination

### PR DESCRIPTION
**Pull request by udhadpa**
Hello All,
Please review my PR for pagination (DES-544).
- Props in dispatch event are changed,so had to build a new config method
- Re-do old tests to match the new styles and functionality
- Added styling tests and new functionality tests to standard pagination
- Added tests for new type  - Compact Pagination.
Do let me know your feedback. 
@umahaea @briandunnpearson @wyseguyonline @mhomolak 
Thanks

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,11 +27,11 @@ before_script:
 - python -m SimpleHTTPServer &>/dev/null &
 
 script:
-- export component=elements-sdk
-- export feature_branch=master
+- export component=pagination
+- export feature_branch=DES-544
```
Comment (briandunnpearson) [inline .travis.yml]: Not sure you want to save this as "DES-544" instead of master.

File: src/test/java/standAloneTests/PaginationTest.java [hunk 2]
```diff
@@ -93,384 +93,504 @@ private void afterMethod() throws IOException, InterruptedException {
...
+            throw new SkipException("Hover operation not yet supported in firefox/safari/ie browser drivers");
+        }
+        String[] detailProperties = new String[]{"elementId", "pagination-target"};
+        String[] propsPropertiesList = new String[]{"activePage", "1", "pages", "10"};
+        String[] expBorderBottoms = new String[]{"2px", "solid", commonUtils.hex2Rgb("#6a7070"), commonUtils.hex2RgbWithoutTransparency("#6a7070")};
+        setConfigAndLaunch(detailProperties, propsPropertiesList);
+        Thread.sleep(1000);
+        for (int i = 2; i < 5; i++) {
+            commonUtils.hoverOnElement(By.cssSelector(paginationPgObj.getSelectedPageSpan(i)));
+            Thread.sleep(1000);
```
Comment (umahaea) [inline src/test/java/standAloneTests/PaginationTest.java]: same

File: src/test/java/standAloneTests/PaginationTest.java [hunk 3]
```diff
@@ -93,384 +93,504 @@ private void afterMethod() throws IOException, InterruptedException {
...
+    @Test(testName = "Page Numbers CSS Test", groups = "desktop-regression")
+    private void PageNumberCSSTest() throws Exception {
+        String[] detailProperties = new String[]{"elementId", "pagination-target"};
+        String[] propsPropertiesList = new String[]{"activePage", "1", "pages", "10"};
+        String[] expBorderBottoms = new String[]{"2px", "solid", commonUtils.hex2Rgb("#252525"), commonUtils.hex2RgbWithoutTransparency("#252525")};
+        setConfigAndLaunch(detailProperties, propsPropertiesList);
+        Thread.sleep(1000);
+        for (int i = 2; i < 5; i++) {
+            commonUtils.click(By.cssSelector(paginationPgObj.getSelectedPageSpan(i)));
+            Thread.sleep(1000);
```
Comment (umahaea) [inline src/test/java/standAloneTests/PaginationTest.java]: same

File: src/test/java/standAloneTests/PaginationTest.java [hunk 4]
```diff
@@ -93,384 +93,504 @@ private void afterMethod() throws IOException, InterruptedException {
...
-    public void validateEllipsesOnPaginationTest() throws Exception {
+    @Test(testName = "Page Navigation - Standard Pagination Test", groups = {"desktop-regression"})
+    private void pageNavigationStdTest() throws Exception {
+        String[] detailProperties = new String[]{"elementId", "pagination-target"};
+        String[] propsPropertiesList = new String[]{"activePage", "1", "pages", "10"};
+        setConfigAndLaunch(detailProperties, propsPropertiesList);
+        Thread.sleep(1000);
+        for (int i = 2; i < 5; i++) {
+            commonUtils.click(By.xpath(paginationPgObj.getRightNavBtn()));
+            Thread.sleep(1000);
```
[the remark this answers is not in the document]
Comment (udhadpa) [reply src/test/java/standAloneTests/PaginationTest.java]: Ahh good catch.. Din't realize :)

**Comment by udhadpa** [comment]
Got approvals. Good to merge

**Pull request merged**